docs/connecting-data/connecting-to-apis-and-external-data/asana-api.md

```diff
@@ -1,7 +1,217 @@
----
-description: How to Connect DataDistillr to the Asana API
----
+# Connecting to the Asana API
 
-# Asana API
+## Creating an Asana account
+Set up an account with [Asana](https://asana.com/create-account).
 
-Details coming soon!
\ No newline at end of file
+### **Costs**
+There are a few different types of accounts:
+
+- Free
+- $10.99 per user per month billed annually or $13.49 billed monthly
+- $24.99 per user per month billed annually or $30.49 billed monthly
+
+[Details and Comparison](https://asana.com/pricing)
+
+### **Rate Limits**
+Limits are allocated per authorization token. Limits may vary from token to token.
+
+Free account limit: 150 requests per minute.
+Premium account limit: 1500 requests per minute.
+SearchAPI  are limited to 60 requests per minute.
+Duplication endpoints are limited to 5 concurrent jobs.
+
+## How to Connect DataDisillr to Asana
+To set up a data source connection for Asana, you will need to have:
+
+- A [unique name](/connecting-data/connecting-to-apis-and-external-data/asana-api/#name) for your data source connection to be used in queries.
+- The [API key](/connecting-data/connecting-to-apis-and-external-data/asana-api/#api-key) generated through your Asana account.
+
+### Data Source Form
+
+To locate the Asana form, follow the steps in [Connecting Your Data to DataDistillr](../../). When you get to the window to choose the data source type, select API as shown below.
+
+<figure markdown>
+![Select API from the available choices] [image-3]{width=500}
+</figure>
+
+On the API screen, select Asana from the list of API types as shown in the image below.
+
+<figure markdown>
+![Select Asana from API types] [image-4]{width=600}
+</figure>
+
+The following form will appear. Instructions can be found below on how to find the information required to fill each field on the Asana API form.
+
+Once you have filled out all the fields, press the green 'Save' button, and your API will be connected!
+
+<figure markdown>
+![Asana Form] [image-1]{width=600}
+</figure>
+
+### Name
+Enter any name that will help you recognize this data source from within your query window.
+
+!!! info "Acceptable Characters Include"
+
+    - lowercase alphanumeric characters
+    - underscores
+
+### API Key
```

Comment:
Something funky is going on here - the pages do not appear in the graphic they did before

Reply:
Pages 3-6 look ok, but 1 and 2 are on their own and it displays <<<<<<< HEAD throughout